doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -323,7 +350,13 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
```

Comment:
Please also add (it should be the scope of Matter samples I suppose):

KRKNWK-18242: Thermostat sample does support the AUTO system mode
  AUTO system mode is supported in the thermostat device but it is not reflected in the
  in the data model and the feature map indicates that AUTO system mode is not supported.
 
  **Workaround:** Set the feature map to `0x23` in the `thermostat.zap` file and regenerate data model, so that it  indicates that the AUTO mode is supported by thermostat device.

Reply:
Just for v2-6-0?

Reply:
Sorry, starting from 2.5.0 release:

.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0